src/ORM/Table.php

```diff
@@ -2194,7 +2197,7 @@ protected function _saveMany(iterable $entities, $options = []): iterable
             throw $e;
         }
 
-        if ($failed !== null) {
+        if (isset($failed)) {
```

Comment:
You can keep this if you move $failed = null before the try block.

Reply:
Will do that.